cmd/pebble/cmd_run.go

```diff
@@ -49,6 +50,7 @@ func init() {
 	addCommand("run", shortRunHelp, longRunHelp, func() flags.Commander { return &cmdRun{} },
 		map[string]string{
 			"create-dirs": "Create pebble directory on startup if it doesn't exist",
+			"dry":         "Validate plan without starting the pebble daemon.",
```

Comment:
"Attempt to run without actually running"

This keeps it open ended, as we'll want to do more than just validate the plan there.

As a convention trivial, please note the lack of full stop on these sentences.